net-im/spectrum2/files/use_qt5_libcommuni.patch

```diff
@@ -0,0 +1,55 @@
+--- a/CMakeLists.txt	2018-02-07 11:57:43.000000000 +0100
++++ b/CMakeLists.txt	2018-04-04 22:12:33.000000000 +0200
+@@ -200,8 +200,7 @@
+ 	set(Communi_DIR "${CMAKE_SOURCE_DIR}/cmake_modules")
+ 	find_package(Communi)
+ 
+-	INCLUDE(FindQt4)
+-	FIND_PACKAGE(Qt4 COMPONENTS QtCore QtNetwork)
++	FIND_PACKAGE(Qt5 COMPONENTS Core Network)
+ 	# ADD_DEFINITIONS(${SWIFTEN_CFLAGS})
+ 	ADD_DEFINITIONS(-DSUPPORT_LEGACY_CAPS)
+ 	# ADD_DEFINITIONS(-DBOOST_FILESYSTEM_VERSION=2)
+@@ -341,7 +340,6 @@
+ 		message("IRC plugin        : yes")
+ 		include_directories(${QT_QTNETWORK_INCLUDE_DIR})
+ 		include_directories(${IRC_INCLUDE_DIR})
+-		include(${QT_USE_FILE})
+ 	else()
+ 		if(ENABLE_IRC)
+ 			message("IRC plugin        : no (install libCommuni and libprotobuf-dev)")
+--- a/backends/libcommuni/CMakeLists.txt	2018-02-07 11:57:43.000000000 +0100
++++ b/backends/libcommuni/CMakeLists.txt	2018-04-04 22:20:11.000000000 +0200
+@@ -1,14 +1,15 @@
+ cmake_minimum_required(VERSION 2.6)
++find_package(Qt5Network REQUIRED)
+ FILE(GLOB SRC *.cpp)
+ FILE(GLOB HEADERS *.h)
+-QT4_WRAP_CPP(SRC ${HEADERS} OPTIONS -DBOOST_TT_HAS_OPERATOR_HPP_INCLUDED)
+-add_definitions(-DQT_NO_KEYWORDS)
++QT5_WRAP_CPP(SRC ${HEADERS} OPTIONS -DBOOST_TT_HAS_OPERATOR_HPP_INCLUDED)
+ ADD_EXECUTABLE(spectrum2_libcommuni_backend ${SRC})
++include_directories(${Qt5Core_INCLUDE_DIRS} ${Qt5Network_INCLUDE_DIRS})
+ 
+ if (NOT WIN32)
+-	target_link_libraries(spectrum2_libcommuni_backend ${IRC_LIBRARY} Qt4::QtNetwork Qt4::QtCore transport pthread)
++	target_link_libraries(spectrum2_libcommuni_backend ${IRC_LIBRARY} Qt5::Network Qt5::Core transport pthread)
+ else ()
+-	target_link_libraries(spectrum2_libcommuni_backend ${IRC_LIBRARY} Qt4::QtNetwork Qt4::QtCore transport)
++	target_link_libraries(spectrum2_libcommuni_backend ${IRC_LIBRARY} Qt5::Network Qt5::Core transport)
+ endif()
+ INSTALL(TARGETS spectrum2_libcommuni_backend RUNTIME DESTINATION bin)
+ 
+--- a/cmake_modules/CommuniConfig.cmake	2018-02-07 11:57:43.000000000 +0100
```

Comment:
i don't see this anywhere

Reply:
Thanks, bumped to 2.0.8 and updated those patches.